app/src/main/java/com/owncloud/android/ui/fragment/OCFileListBottomSheetDialog.java

```diff
@@ -124,16 +126,23 @@ protected void onCreate(Bundle savedInstanceState) {
 
                     View creatorView = creatorViewBinding.getRoot();
 
-                    creatorViewBinding.creatorName.setText(
-                        String.format(fileActivity.getString(R.string.editor_placeholder),
-                                      fileActivity.getString(R.string.create_new),
-                                      creator.getName()));
-
-                    creatorViewBinding.creatorThumbnail.setImageDrawable(
-                        MimeTypeUtil.getFileTypeIcon(creator.getMimetype(),
-                                                     creator.getExtension(),
-                                                     creatorViewBinding.creatorThumbnail.getContext(),
-                                                     viewThemeUtils));
+                    //for NMC we have different text and icon for Markdown(.md) menu
+                    if (creator.getMimetype().equalsIgnoreCase(MimeTypeUtil.MIMETYPE_TEXT_MARKDOWN)) {
```

Comment:
In my test this now shows "create text document" with new icon, but when clicking on it, it still suggests to create "empty file.md".
(as this is done by server always a Markdown file will be created)

Reply:
Yes that's right clicking action is same we have just changed new text and icon for it.

Reply:
This will be added at run time when the creators have the data. And it is using LinearLayout to add the views.
So if i will add text and icon in xml then i will have to comment the creators logic.